overlord/overlord_test.go

```diff
@@ -58,6 +58,26 @@ type overlordSuite struct {
 
 var _ = Suite(&overlordSuite{})
 
+type ticker struct {
+	tickerChannel chan time.Time
+}
+
+func (w *ticker) tick(n int) {
+	for i := 0; i < n; i++ {
+		w.tickerChannel <- time.Time{}
```

Comment:
(nitpick) a real ticker would return time.Now(), correct? I guess we could do this too.